### PR TITLE
HIP-904: add airdrop claim protos

### PR DESCRIPTION
**Pull request by vtronkov**
**Description**:
Proto changes supporting TokenClaimAirdrop

**Related issue(s)**:

Fixes https://github.com/hashgraph/hedera-services/issues/14201

### CODE REVIEW

**Review by Neeharika-Sompalli — COMMENTED** [review]
LGTM! Except Joseph's comments. Also, do we expect any response codes to be added ?

**Comment by vtronkov** [comment]
> LGTM! Except Joseph's comments. Also, do we expect any response codes to be added ?

Per this document I didn't find new response codes: https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/docs/design/services/token-service/airdrops/token-claim-airdrop.md

**Review by jsync-swirlds — APPROVED** [review]

**Review by edwin-greene — APPROVED** [review]

File: services/basic_types.proto [hunk 1]
```diff
@@ -1745,3 +1750,32 @@ message PendingAirdropValue {
...
+    /**
+     * A list of one or more pending airdrop identifiers.
+     * <p>
+     * This transaction MUST be signed by the account identified by
+     * the `receiver_id` for each entry in this list.<br/>
+     * This list MUST contain between 1 and 10 entries, inclusive.<br/>
+     * This list MUST NOT have any duplicate entries.
+     */
+    repeated PendingAirdropId pending_airdrops = 1;
+}
```
Comment (jsync-swirlds) [inline services/basic_types.proto:1781]: The general pattern for Services is to add a new proto file for each transaction (e.g. `token_create`, `token_reject`, `token_update`, etc...).
This should _probably_ be in a file `token_claim_airdrop.proto`.

Comment (vtronkov) [reply services/basic_types.proto]: Fixed 👍

**Pull request merged**